src/lsptoolshost/roslynProtocol.ts

```diff
@@ -221,6 +221,16 @@ export interface ProjectNeedsRestoreName {
     projectFilePaths: string[];
 }
 
+export interface CopilotRelatedDocumentsParams extends lsp.WorkDoneProgressParams, lsp.PartialResultParams {
+    _vs_textDocument: lsp.TextDocumentIdentifier;
+    position: lsp.Position;
+}
+
+export interface CopilotRelatedDocumentsReport {
+    _vs_resultId: string | null;
```

Comment:
I noticed we have a resultId in the server response, but its not used, and it is not present in the client request either.  That seems like a bug - otherwise we're going to be recomputing this information on every single request.

Reply:
created a PR to remove resultID as discussed offline:
https://github.com/dotnet/roslyn/pull/75176